src/test/metrics_flow_test.cpp

```diff
@@ -852,6 +897,7 @@ std::string MetricFlowTest::prepareConfigContent() {
            R"(",")" + METRIC_NAME_REQUESTS_ACCEPTED +
            R"(",")" + METRIC_NAME_REQUESTS_REJECTED +
            R"(",")" + METRIC_NAME_RESPONSES +
+           R"(",")" + METRIC_NAME_GRAPH_ERROR +
            R"("]
             }
         },
```

Comment:
missing grpc tests, similar tests with grpc setup should be already somewhere in our suite